src/utrfx/util.py

```diff
@@ -8,14 +8,17 @@
 
 def download_fasta_from_ensembl(transcript_id: str, timeout: float = 30.,) -> str:
```

Comment:
It seems to me that the function would be better named `fetch_cdna_from_ensembl` (or something similar). FASTA is an implementation detail and we should not be mentioning it.